doc/connectivity/bluetooth/autopts/autopts-linux.rst

```diff
@@ -338,15 +369,17 @@ At the first run, when Windows asks, enable connection through firewall:
    :align: center
 
 Troubleshooting
-================
+****************
 
-- "After running one test, I need to restart my Windows virtual machine to run another, because of fail verdict from APICOM in PTS logs."
+After running one test, I need to restart my Windows virtual machine to run another, because of fail verdict from APICOM in PTS logs
+====================================================================================================================================
 
 It means your virtual machine has not enough processor cores or memory. Try to add more in
 settings. Note that a host with 4 CPUs could be not enough with VirtualBox as hypervisor.
 In this case, choose rather VMWare Workstation.
 
-- "I cannot start autoptsserver-zephyr.py. I always got error:"
+I cannot start autoptsserver-zephyr.py. I always got error
```

Comment:
I now you're not the original author but maybe updating the sentence is worth it

```suggestion
I cannot start autoptsserver-zephyr.py. I always get a Python error similar to the one in the following screenshot
```

Reply:
That's fine, I just feel like it's strange to have an image out of nowhere without introducing it

Reply:
That's previous content btw - Nothing I added. I only modified the header ;)

Reply:
Yes, that's why it's was more of a general comment. On the previous version there was `:` which kinda made the link between the sentence and the image. But again, that's just general comment, feel free to ignore :)